### ISSUE TITLE
Primary obstructive megaureter

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6658]
**Preferred term label:**
Primary obstructive megaureter

**Synonyms**
Primary megaureter

**Definition (free text, please give PubMed ID)**
Result of obstruction or an adynamic ureteral segment in the ureterovesical junction. 
doi: https://doi.org/10.1155/2013/198592

**Parent term (use hpo.jax.org/app)**
Hydroureter

**Comment by pnrobinson** [comment]
@johannaest There are a lot of different terminologies and usages, but probably megaureter is best viewed as a synonym of hydroureter. Probably we should adopt the classification sketched out here
https://www.sciencedirect.com/topics/medicine-and-dentistry/megaureter
-- thoughts?

**Comment by johannaest** [comment]
Yes, I would add Megaureter as a synonym of hydroureter.  And Primary obstructive megaureter as a type of megaureter/hydroureter (so hydroureter as the parent term).

**Comment by pnrobinson** [comment]
added
